scripts/upgrade.sh

```diff
@@ -102,6 +102,22 @@ verify_masters_versions() {
   done
 }
 
+replace_config_file(){
+KUBECTL_BIN="/opt/k3s kubectl"
+CONFIG_FILE="/host/etc/rancher/k3s/config.yaml"
+LABELS=$(${KUBECTL_BIN} get nodes $HOSTNAME  --show-labels | tail -n +2  | awk '{print $6}' )
+LIST=$(echo $LABELS | tr "," "\n" | tr "=" "-")
```

Comment:
I'm confused by the `tr "=" "-"` bit - won't this mangle the labels?

Reply:
i had this originally cause secrets are named with label names, and it cant include `=` in it